core/src/main/kotlin/com/strumenta/kolasu/model/Naming.kt

```diff
@@ -57,20 +63,15 @@ class ReferenceByName<N>(val name: String, initialReferred: N? = null) : Seriali
         }
     }
 
-    val resolved: Boolean
-        get() = referred != null
-
     override fun hashCode(): Int {
         return name.hashCode() * (7 + if (resolved) 2 else 1)
     }
 
     override fun equals(other: Any?): Boolean {
```

Comment:
We should consider also `identifie` in `equals` (and possibly in `hashCode` too)